.github/workflows/build.yml

```diff
@@ -312,26 +322,64 @@ jobs:
     steps:
       - name: Setup | Checkout
         uses: actions/checkout@v3
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
+        if: ${{ needs.release_please.outputs.release_created != 'true' }}
+        with:
+          path: artifacts
+
+      - name: Copy script - bucket
+        if: ${{ needs.release_please.outputs.release_created == 'true' }}
+        run: |
+          mkdir -p bucket
+          sed -e "s#__VERSION__#${{ needs.release_please.outputs.version_name }}" install/install.sh > bucket/install.sh
+      - name: Copy PR scripts - bucket
+        if: ${{ needs.release_please.outputs.release_created != 'true' }}
+        run: |
+          mkdir -p bucket
+          rm -rf bucket/artifacts
+          mkdir -p bucket/artifacts
+          find artifacts -type f -exec cp {} bucket/artifacts \;
```

Comment:
`cp artifacts/* bucket/artifacts/`

Reply:
The tree is artifacts/ARTIFACT/ARTIFACT.tar.gz ....

Reply:
ah ok, then yes, the `find` command is better suited